classes/class-wc-connect-nux.php

```diff
@@ -83,5 +97,300 @@ public function register_add_service_to_zone_pointer( $pointers ) {
+					add_action( 'wp_ajax_woocommerce_services_get_jetpack_connect_url',
+						array( $this, 'ajax_get_jetpack_connect_url' )
+					);
+					wp_enqueue_style( 'wc_connect_banner' );
+					add_action( 'admin_notices', array( $this, 'show_banner_before_connection' ), 9 );
+					break;
+				case self::JETPACK_CONNECTED:
+					// Has the after-connection notice been dismissed already?
+					if ( WC_Connect_Options::get_option( self::SUCCESS_BANNER_IS_DISMISSED ) ) {
+						break;
+					}
+					wp_enqueue_style( 'wc_connect_banner' );
+					add_action( 'admin_notices', array( $this, 'show_banner_after_connection' ) );
+					break;
+			}
+		}
+
+		public function show_banner_before_connection() {
+			if ( ! $this->should_display_nux_notice_on_screen( get_current_screen() ) ) {
+				return;
+			}
+
+			// Remove Jetpack's connect banners since we're showing our own.
+			if ( class_exists( 'Jetpack_Connection_Banner' ) ) {
+				$jetpackBannerInstance = Jetpack_Connection_Banner::init();
+				remove_action( 'admin_notices', array( $jetpackBannerInstance, 'render_banner' ) );
+				remove_action( 'admin_notices', array( $jetpackBannerInstance, 'render_connect_prompt_full_screen' ) );
+			}
+
+			// Make sure to show the after-connection success message even after Jetpack disconnect.
+			WC_Connect_Options::delete_option( self::SUCCESS_BANNER_IS_DISMISSED );
+
+			$jetpack_status = $this->get_jetpack_install_status();
+
+			$button_text = __( 'CONNECT >', 'woocommerce-services' );
+
+			$image_url = plugins_url( 'images/nux-printer-laptop-illustration.png', dirname( __FILE__ ) );
+
+			switch ( $jetpack_status ) {
+				case self::JETPACK_UNINSTALLED:
+					$button_text = __( 'Install Jetpack and CONNECT >', 'woocommerce-services' );
+					break;
+				case self::JETPACK_INSTALLED:
+					$button_text = __( 'Activate Jetpack and CONNECT >', 'woocommerce-services' );
+					break;
+			}
+
+			$default_content = array(
+				'title'           => __( 'Connect your store to activate WooCommerce Shipping', 'woocommerce-services' ),
+				'description'     => __( "WooCommerce Shipping is almost ready to go! Once you connect your store you'll be able to access discounted rates and printing services for USPS and Canada Post from your dashboard (fewer trips to the post office, winning).", 'woocommerce-services' ),
+				'button_text'     => $button_text,
+				'image_url'       => $image_url,
+				'should_show_jp'  => true,
+			);
+
+			$base_location = wc_get_base_location();
+			$country = isset( $base_location['country'] )
+				? $base_location['country']
+				: '';
+			switch ( $country ) {
+				case 'CA':
+					$localized_content = array(
+						'description'     => __( "WooCommerce Shipping is almost ready to go! Once you connect your store you'll be able to show your customers live shipping rates when they check out.", 'woocommerce-services' ),
+					);
+					break;
+			}
+
+			$this->show_nux_banner( wp_parse_args( $localized_content = null, $default_content ) );
```

Comment:
I wouldn't want to do that since there are different defaults for `show_banner_after_connection()` and `show_banner_before_connection()`, but they both use `show_nux_banner()`.